subprojects/resources-s3/src/integTest/groovy/org/gradle/integtests/resource/s3/ivy/IvyPublishS3IntegrationTest.groovy

```diff
@@ -17,9 +17,12 @@
 package org.gradle.integtests.resource.s3.ivy
 
 import org.gradle.api.publish.ivy.AbstractIvyPublishIntegTest
+import org.gradle.integtests.resource.s3.fixtures.S3IntegrationTestPrecondition
 import org.gradle.integtests.resource.s3.fixtures.S3Server
 import org.junit.Rule
+import spock.lang.Requires
 
+@Requires({ S3IntegrationTestPrecondition.fulfilled })
```

Comment:
Why can't we test it on Java8 + embedded?

Reply:
Because we have JAXB on the classpath twice: Once from the JDK and once from the additional JAR. I tried excluding the JAR from the test classpath but then every test that uses the `GradleExecutor` fails because we throw an exception for missing JARs in `DefaultModuleRegistry`.

Reply:
Got it.